dev/com.ibm.ws.security.token.ltpa/resources/com/ibm/ws/security/token/ltpa/internal/resources/LTPAMessages.nlsprops

```diff
@@ -48,12 +48,28 @@ LTPA_CONFIG_ERROR=CWWKS4106E: LTPA configuration error. Unable to create or read
 LTPA_CONFIG_ERROR.explanation=The LTPA keys could not be read or created at the specified location.
 LTPA_CONFIG_ERROR.useraction=Ensure the location is accessible by the server process. Any FFDCs might indicate a fatal error in generating or loading the keys.
 
-LTPA_KEYS_TO_LOAD=CWWKS4107A: The LTPA keys file or configuration were modified. The LTPA keys will be reloaded from the file: {0}
-LTPA_KEYS_TO_LOAD.explanation=The LTPA keys need to be reloaded because there was a change to the LTPA keys file or the configuration.
-LTPA_KEYS_TO_LOAD.useraction=Ensure that all servers participating in Single-Sign-On use the same LTPA keys and password.
+LTPA_KEYS_TO_LOAD=CWWKS4107A: The LTPA keys files are either created, deleted, or modified: {0}, or the LTPA configuration was modified. All the LTPA keys will be reloaded.
```

Comment:
1. is {0} a list of API key files?
2. if the LTPA configuration was modified but no LTPA keys files were created, deleted, or modified, what is included in {0}?

Reply:
It's a list of LTPA keys files.

Reply:
ok, posting a new suggestion.